translation/swift/Translation/AppDelegate.swift

```diff
@@ -0,0 +1,181 @@
+//
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Update all License headers to : `// Copyright 2019 Google LLC` 

Drop all periods, and "All Rights Reserved."

Reply:
Updated.